service/src/main/kotlin/app/cash/backfila/service/BackfilaDevelopmentService.kt

```diff
@@ -54,7 +96,7 @@ fun main(args: Array<String>) {
                   mapOf("backfila-001" to DataSourceClusterConfig(
                       writer = DataSourceConfig(
                           type = DataSourceType.MYSQL,
-                          database = "backfila_development",
+                          database = "backfila_test",
```

Comment:
Was this just wrong before?

Reply:
oops will revert, I was testing with this